js/reference.js

```diff
@@ -4014,6 +4014,29 @@
             return this._cascadingDeletedItems;
         },
 
+        /**
+         * If prefill object is defined and has the required attributes, will return
+         * a Prefill object with the necessary objects used for a association modal picker
+         *
+         * @type {ERMrest.Prefill}
+         */
+        get prefill() {
+            if (this._prefill === undefined) {
+                // TODO: error with message "Call computePrefill first"
```

Comment:
I wasn't sure what was the best way to handle this. My TODO was intended to remind me to discuss this with you.